source/GameInterface/Services/MobileParties/Patches/HourlyPartyTickServerPatch.cs

```diff
@@ -0,0 +1,26 @@
+﻿using GameInterface.Services.MobileParties.Extensions;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.Party;
+
+namespace GameInterface.Services.MobileParties.Patches;
+
+[HarmonyPatch(typeof(CampaignEvents))]
+internal class HourlyPartyTickServerPatch
+{
+    [HarmonyPrefix]
+    [HarmonyPatch("HourlyTickParty")]
+    private static bool HourlyTickPartyPrefix(ref MobileParty mobileParty)
+    {
+        if (ModInformation.IsServer)
+        {
+            if (mobileParty.IsPartyControlled())
+            {
+                return true;
+            }
+            return false;
+        }
+        return false;
+    }
```

Comment:
All ticking should only run on the server
```suggestion
    private static bool HourlyTickPartyPrefix(ref MobileParty mobileParty) => ModInformation.IsServer;
```